presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -327,6 +327,7 @@ public final class SystemSessionProperties
     public static final String INLINE_PROJECTIONS_ON_VALUES = "inline_projections_on_values";
     public static final String INCLUDE_VALUES_NODE_IN_CONNECTOR_OPTIMIZER = "include_values_node_in_connector_optimizer";
     public static final String SINGLE_NODE_EXECUTION_ENABLED = "single_node_execution_enabled";
+    public static final String EXPRESSION_OPTIMIZER_INSTANCE = "expression_optimizer_instance";
```

Comment:
maybe call this "expression_optimizer_name"? instance makes it sound like we're providing the object itself.

Reply:
Agreed, done.